GraknConsole.java

```diff
@@ -201,6 +208,9 @@ private void runQuery(GraknClient.Transaction tx, String queryString) {
             printer.error(e.getMessage());
             return;
         }
+
+        ExecutorService executorService = Executors.newSingleThreadExecutor();
```

Comment:
needed to run iterator.next() in a separate thread, so in the main thread you can cancel it